clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -1492,6 +1501,20 @@ public static <E> Set<E> diff(final Supplier<Set<E>> constructor, final Set<E> l
         return result;
     }
 
+    /**
+     * @param set Source set.
+     * @param toRemove Elements to remove.
+     * @return {@code set} copy without {@code toRemove} elements.
+     * @param <T> Element type.
+     */
+    @SuppressWarnings("unchecked")
+    public static <T> Set<T> minus(Set<T> set, T...toRemove) {
```

Comment:
Let's not add methods in the client module that are only used in tools. This can be moved to ToolsUtils.